app/assets/javascripts/list.js

```diff
@@ -7,25 +7,29 @@ $(document).ready(function() {
   });
 
   function changeStep() {
-    document.querySelector(`#${current}`).classList.remove('current');
-    document.querySelector(`#${this.id}`).classList.add('current');
-    document.querySelector('#content').innerHTML = allSteps[this.id];
-    current = this.id;
-  }
-
+    var id = this.id;
+    var currentDescription = document.querySelector(`.${current}`);
+    var newDescription = document.querySelector(`.${id}`);
```

Comment:
I'm really liking how you cleaned all this up by moving the HTML away from the JS. 👍